src/main/js/templates/plugin-manager/available.hbs

```diff
@@ -61,13 +61,26 @@
             {{/if}}
         </td>
         {{#if this.releaseTimestamp }}
-            <td style="width: 25%" data="{{ this.releaseTimestamp.iso8601 }}">
+            <td style="width: 20%" data="{{ this.releaseTimestamp.iso8601 }}">
                 <time datetime="{{ this.releaseTimestamp.iso8601 }}">
                     {{ this.releaseTimestamp.displayValue }}
                 </time>
             </td>
         {{else}}
-            <td style="width: 25%"></td>
+            <td style="width: 20%"></td>
         {{/if}}
+        <td style="width: 15%">
+            <div>
+              {{#if this.healthScore }}
+                <a href="{{ this.wiki }}/healthScore"
+                   class="jenkins-healthScore--badge"
```

Comment:
Here is missing the `jenkins-healthScore--[top|middle|bottom]` class I think

Reply:
Yes definitively. See https://github.com/jenkinsci/jenkins/pull/10351#issuecomment-2715026860.
But I might have found a trick.